cmd/minishift/cmd/util/util_test.go

```diff
@@ -48,3 +48,36 @@ func TestIsValidProfileName(t *testing.T) {
 		}
 	}
 }
+
+func TestRunAndGetExitCode(t *testing.T) {
+	testData := []struct {
+		command  string
+		exitCode int
+	}{
+		{"blahh", 1},
+		{"make", 2},
```

Comment:
Why do you expect '2' here? How do you know whether or not 'make' is installed?

Reply:
when make is run without there being a `Makefile` it returns error code 2, well to run the tests we do `make test` so i figured `make` has to be installed... is that a bad assumption?